### DISCUSSION TITLE
How to use the login function

### DISCUSSION THREAD

**Discussion by jiaoting** [rustdesk/rustdesk#2143] [Q&A]
![image](https://user-images.githubusercontent.com/12553424/202090799-824d61c7-ba71-471a-8510-baccc6ec304d.png)
这块的功能是啥？我用自建服务器，也能使用吗？

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/1337

**Reply by jiaoting** [reply]
thx~另外，ios和ipad端建议增加屏幕旋转，这样可以充分利用屏幕的宽度
